bindings/swift/Cargo.toml

```diff
@@ -15,4 +15,7 @@ futures =  { version = "0.3.25", default-features = false }
 once_cell = { version = "1.16.0", default-features = false }
 serde = { version = "1.0.147", default-features = false, features = [ "derive" ] }
 serde_json = { version = "1.0.88", default-features = false }
-tokio = { version = "1.22.0", default-features = false}
+tokio = { version = "1.22.0", default-features = false }
+
```

Comment:
Remove the new line and order them within the rest

Reply:
Sad times, i thought logging separate :)